regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -58,6 +61,7 @@ var PreambleView = ChildView.extend({
     this.$write = this.$el.find('#preamble-write');
     var section = this.$read.closest('section').attr('id');
     var docId = section.split('-')[0];
+    this.PreambleHeadView = new PreambleHeadView();
```

Comment:
We'll probably want to tear this down in `PreambleView.remove` so that we don't accumulate lots of dead views. We make a new `PreambleView` and destroy the existing one (if any) every time the user ajax-navigates preambles, so we need to clean up child views on `remove`.